realm/src/main/java/io/realm/Realm.java

```diff
@@ -1623,10 +1615,6 @@ void setVersion(long version) {
         }
     }
 
-    private <E extends RealmObject> boolean isObjectInRealm(E object) {
```

Comment:
It is a tradeoff. Now the method is only used from the proxy classes, so I would rather save the method call + package protected method in Realm vs. slightly longer code in a autogenerated class.

It is not something I feel strongly about, but we do have a lot of package protected methods which makes it hard to move more specialised API's to other packages (like .dynamic or .migration). In this case I didn't think it was worth it to keep a package protected method.